src/app/containers/CpsRecommendations/index.jsx

```diff
@@ -0,0 +1,203 @@
+import { StoryPromoLi, StoryPromoUl } from '@bbc/psammead-story-promo-list';
+import { C_LUNAR, C_EBON, C_GHOST } from '@bbc/psammead-styles/colours';
+import Image from '@bbc/psammead-image';
+import { getDoublePica } from '@bbc/gel-foundations/typography';
+import { getSansRegular } from '@bbc/psammead-styles/font-styles';
+import {
+  GEL_SPACING,
+  GEL_SPACING_HLF,
+  GEL_SPACING_DBL,
+} from '@bbc/gel-foundations/spacings';
+
+import { ServiceContext } from '#contexts/ServiceContext';
+import useToggle from '#hooks/useToggle';
+import CpsOnwardJourney from '../CpsOnwardJourney';
+import Grid from '../../components/Grid';
+import data from './recommendations.ltr.json';
+
+const GEL_SPACING_DBL_MINUS_HLF = '0.75rem';
+
+const StoryPromoWrapper = styled(StoryPromo)`
+  display: grid;
+  margin: ${GEL_SPACING_HLF} ${GEL_SPACING_DBL};
+  background-color: ${C_GHOST};
+`;
+
+const CustomSectionLabel = styled(SectionLabel).attrs(props => ({
+  'aria-labelledby': props.labelId,
+}))`
+  ${({ script }) => script && getDoublePica(script)};
+  ${({ service }) => getSansRegular(service)};
+  color: ${C_EBON};
+  margin-top: 0;
+  background-color: ${C_LUNAR};
+`;
+
+const SectionLabelWrapper = styled.div`
+  padding: ${GEL_SPACING_DBL};
+`;
+
+const WsojWrapper = styled.div`
+  background-color: ${C_LUNAR};
+  padding-bottom: ${GEL_SPACING};
+`;
+
+const StyledStoryPromoLi = styled(StoryPromoLi)`
+  /* Override various paddings set by StoryPromoLi */
+  margin: -${GEL_SPACING_DBL_MINUS_HLF} 0;
+`;
+
+const getPromoItemProps = item => ({
+  src: path(['indexImage', 'path'], item),
+  href: path(['locators', 'assetUri'], item),
+  headline: path(['headlines', 'headline'], item),
+  alt: path(['indexImage', 'altText'], item),
+});
+
+const toStoryPromoItem = ({ assetUri, shortHeadline, imageHref }) => ({
+  headlines: {
+    headline: shortHeadline,
+  },
+  locators: {
+    assetUri,
+  },
+  indexImage: {
+    path: imageHref,
+    altText: 'Image Alt text', // This value would be changed as it's on our list of improvements.
+  },
+  uri: assetUri,
+});
+
+const transformItems = items => items.map(toStoryPromoItem);
+
+const CpsRecommendations = ({ parentColumns, items }) => {
+  const { script, service, dir, recommendations } = useContext(ServiceContext);
+  const { enabled } = useToggle('cpsRecommendations');
+
+  const { hasStoryRecommendations } = recommendations;
+
+  if (!hasStoryRecommendations || !enabled) return null;
+
+  const singleTransform = item => {
+    const { src, href, headline, alt } = getPromoItemProps(item);
+
+    const Img = <Image alt={alt} src={src} />;
+
+    const Info = (
+      <Headline script={script} service={service} promoHasImage as="span">
+        <Link href={href}>{headline}</Link>
+      </Headline>
+    );
+
+    return (
+      <Grid
+        columns={{
+          group0: 1,
+          group1: 1,
+          group2: 1,
+          group3: 1,
+          group4: 1,
+          group5: 1,
```

Comment:
Add a story to show when there is one recommendation

Reply:
It looks like this has been addressed.